src/equations/compressible_euler_3d.jl

```diff
@@ -974,7 +974,7 @@ end
 
   # convert to entropy `-rho * s` used by Hughes, France, Mallet (1986)
   # instead of `-rho * s / (gamma - 1)`
-  V1,V2,V3,V4,V5 = w * (gamma-1) 
+  V1,V2,V3,V4,V5 = w .* (gamma-1) 
```

Comment:
```suggestion
  V1, V2, V3, V4, V5 = w .* (gamma-1) 
```
For consistency with 1D/2D and elsewhere